docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
+telescopes, and calibration devices.
+The management, definition, derivation, verification, and validation of the simulation model is central to the functionality of simtools.
```

Comment:
I would add to this that all parameters are to be found in the database and are controlled by version names (e.g. `prod5`). I would already quickly mention here that the structure of the parameter is strictly defined by the schema files found in `simtools/schema`

Reply:
OK - added a sentence to this first section.